### PR TITLE
 Updating the building and testing steps within the README of the repo.

### PR DESCRIPTION
**Pull request by ahsonkhan**
Updated the steps to be more accurate. Added some clarifying details from recent changes to the wiki: https://github.com/dotnet/corefxlab/wiki

cc @karelz, @dotnet/corefxlab-contrib, @adamsitnik, @MisinformedDNA

### CODE REVIEW

**Comment by adamsitnik** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -117,14 +117,30 @@ There are many .NET related projects on GitHub.
...
+   - The `corefxlab.sln` solution (which contains all the active source and test projects) is built
+   - All the unit tests witin the test projects (that live inside the `tests` folder) are executed.
+      - To skip this step, add `-SkipTests true` as an argument (`build.cmd` only)
+2. After you have have run `build` at least once, you can open the `corefxlab.sln` solution file in [Visual Studio 2017](https://www.visualstudio.com/downloads/) (Community, Professional, or Enterprise), on Windows. Make sure you have the .NET Core workload installed (see [corefx windows build instructions](https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md) for more details). Also, make sure to add the `dotnetcli` folder path to your system path environment variable. If you are using VS Code, see https://aka.ms/vscclrdogfood.
+   - If you cannot change the system path, download or install the [new version of the .NET CLI](https://github.com/dotnet/cli#installers-and-binaries) for your operating system at the default global location `C:\Program Files\dotnet`, for VS to reference.
```
Comment (MisinformedDNA) [inline README.md]: This is not a complete sentence. "If" doesn't have a "then".

Comment (MisinformedDNA) [reply README.md]: While then is not required, it helps avoid ambiguity. I was actually reading the sentence as

> If you cannot change the system path, download or install the new version of the .NET CLI for your operating system at the default global location `C:\Program Files\dotnet`, [then] for VS to reference.

I think you should at least remove the comma before "for VS to reference".

Comment (ahsonkhan) [reply README.md]: https://github.com/dotnet/corefxlab/pull/2309

**Pull request merged**